src/PanZoom.js

```diff
@@ -714,16 +793,4 @@ class PanZoom extends React.Component<Props> {
   }
 }
 
-PanZoom.defaultProps = {
```

Comment:
I would prefer to keep `defaultProps` outside of the class. There is no change defining those inside the class or outside, but if we want to move on and use React Hooks, we will have to have it outside.

Reply:
@mnogueron I did this because otherwise flow would complain and as I understood you need to do this for receiving the correct type-checks: https://github.com/facebook/flow/issues/1660#issuecomment-386619834

Reply:
Good catch then 👍